lib/ecmascript.ts

```diff
@@ -603,12 +617,7 @@ export function ParseTemporalTimeString(isoString: string) {
     millisecond = ToIntegerOrInfinity(fraction.slice(0, 3));
     microsecond = ToIntegerOrInfinity(fraction.slice(3, 6));
     nanosecond = ToIntegerOrInfinity(fraction.slice(6, 9));
-    annotations = match[14];
-    for (const [, critical, key, value] of annotations.matchAll(PARSE.annotation)) {
-      if (key !== 'u-ca' && critical === '!') {
-        throw new RangeError(`Unrecognized annotation: !${key}=${value}`);
-      }
-    }
+    processAnnotations(match[14]); // ignore found calendar
```

Comment:
This is being done for the checking side-effects, right? Can you please be a bit more explicit about that in a comment (so it doesn't get cleaned up on accident)?